gunicorn.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gunicorn todo_app.app:app --preload -b 0.0.0.0:80
```

Comment:
Sorry, just saw this. This is also needs to be fixed for the core exercise. You are telling gunicorn to look inside todo_app/app.py for a variable called `app`.

That doesn't exist.

This command should be `gunicorn "todo_app.app:create_app()" ...etc`

The quotation marks prevent bash from interpreting the brackets as special characters.